### DISCUSSION TITLE
2024/01/25/wo-cao-zhe-me-duo-liang-xin-ruan-jian-du-zai-github-shang-ni-que-da-du-da-bu-kai/

### DISCUSSION THREAD

**Discussion by giscus** [worker2011/worker2011.github.io#1] [Announcements]
# 2024/01/25/wo-cao-zhe-me-duo-liang-xin-ruan-jian-du-zai-github-shang-ni-que-da-du-da-bu-kai/

GitHub 为何物？ Git 是一个代码管理工具。GitHub 上自带 Git，可以上传代码，上面囤积了不少高质量开源项目，比如你谷的 CYaRon。

http://localhost:439/2024/01/25/wo-cao-zhe-me-duo-liang-xin-ruan-jian-du-zai-github-shang-ni-que-da-du-da-bu-kai/

**Comment by worker2011** [comment]
giscus 评论测试

Markdown **加粗**

Markdown ~~删除线~~

```cpp
inline int tst(int a, int b) {
	return a + b;
}
// Markdown 代码块+代码高亮功能测试
```

$2 ^ {10} = 1024$ 功能测试
